services/spar/src/Spar/Run.hs

```diff
@@ -90,19 +98,26 @@ runServer sparCtxOpts = do
         . Bilge.port (sparCtxOpts ^. to galley . epPort)
         $ Bilge.empty
   let wrappedApp
-        = WU.catchErrors sparCtxLogger mx
+        = WU.heavyDebugLogging heavyLogOnly logLevel sparCtxLogger
         . promthRun
+        . WU.catchErrorsException sparCtxLogger [Left mx]
+          -- error 'Response's not thrown as exceptions are logged in 'renderSparErrorIO'
```

Comment:
Given this comment I would expect to see `renderSparErrorIO` in the middleware. Which middleware is that in?

Reply:
updated the comment.